charts/div-vs/values.preview.template.yaml

```diff
@@ -0,0 +1,8 @@
+java:
+    environment:
+        REFORM_ENVIRONMENT: "aat"
```

Comment:
Should this be `preview`? if not maybe we can move it to `values.yaml`

Reply:
Since preview should point to aat I think the reform environment should be aat.
This will change for other envs however, so I think its better to leave it in the template.yaml.

Reply:
👍